talisker/endpoints.py

```diff
@@ -45,26 +46,46 @@ def force_unicode(s):
 
 
 @module_cache
-def networks():
-    networks = os.environ.get('TALISKER_NETWORKS', '').split()
-    return [ip_network(force_unicode(n)) for n in networks if n]
+def get_networks():
+    network_tokens = os.environ.get('TALISKER_NETWORKS', '').split()
+    network_list = []
+    for network in network_tokens:
+        network_list.append(ip_network(force_unicode(network)))
```

Comment:
Good point, that was refactored and then re-refactored, I actually want to leave it alone now. Will do.